docs/user-guide/workflow.ipynb

```diff
@@ -139,6 +139,36 @@
     "workflow = CorrectionWorkflow()\n",
     "workflow.visualize(results, graph_attr={\"rankdir\": \"LR\"})"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is also a half-polarized version of the workflow, in case only a polarizer is used:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ess.polarization.correction import CorrectionWorkflow\n",
+    "\n",
+    "results = (\n",
+    "    pol.HalfPolarizedCorrectedData[pol.Up],\n",
+    "    pol.HalfPolarizedCorrectedData[pol.Down],\n",
+    ")\n",
```

Comment:
Here, `pol.Up` and `pol.Down` stands for the polarizer spin, or for the "dummy analyzer spin"?
As they are part of `pol.HalfPolarizedCorrectedData` ( in correction.py) and defined via `up=p_up` and `down=p_down`, its the polarizer I think? I was just wondering about where at all the "dummy value for the spin of the analyzer that is not present." (code line 170) is used.

Reply:
What you see here is *not* the dummy value. The dummy value is just internal, so we can reuse the workflow graph (and I think you need it when setting the input data).

Reply:
Ok!